src/meson.build

```diff
@@ -0,0 +1,84 @@
+gnotime_cfg_data = configuration_data()
+gnotime_cfg_data.set_quoted('DATADIR', get_option('prefix') / get_option('datadir') / 'gnotime')
+gnotime_cfg_data.set_quoted('GETTEXT_PACKAGE', 'gnotime-2.0')
+gnotime_cfg_data.set_quoted('GNOME_ICONDIR', 'NONE' / get_option('datadir') / 'pixmaps')
+gnotime_cfg_data.set_quoted('GNOMELOCALEDIR', get_option('prefix') / get_option('localedir'))
+gnotime_cfg_data.set_quoted('GTTDATADIR', get_option('prefix') / get_option('datadir') / 'gnotime')
+gnotime_cfg_data.set_quoted('GTTGLADEDIR', get_option('prefix') / get_option('datadir') / 'gnotime')
+gnotime_cfg_data.set_quoted('LIBDIR', get_option('prefix') / get_option('libdir'))
+gnotime_cfg_data.set_quoted('PACKAGE', 'gnotime')
+gnotime_cfg_data.set_quoted('PREFIX', get_option('prefix'))
+gnotime_cfg_data.set_quoted('SYSCONFDIR', get_option('sysconfdir'))
+gnotime_cfg_data.set_quoted('VERSION', meson.project_version() + gtt_version_suffix)
+configure_file(
+  configuration: gnotime_cfg_data,
+  output: 'config.h',
+)
+
+gnotime_deps = [
+  gconf_dep,
+  glib_dep,
+  gtk_dep,
+  gtk_html_dep,
+  guile_dep,
+  libglade_dep,
+  libgnome_dep,
+  libgnomeui_dep,
+  libxml_dep,
+  qof_dep,
+  x11_dep,
+  xscrnsaver_dep,
+]
+if dbus_glib_dep.found()
+  dbus_glib_arg = '-DWITH_DBUS=1'
+  gnotime_deps += dbus_glib_dep
+else
+  dbus_glib_arg = '-DWITH_DBUS=0'
+endif
+
+gnotime_srcs = files(
+  'active-dialog.c',
+  'app.c',
+  'dbus.c',
+  'dialog.c',
+  'err.c',
+  'err-throw.c',
+  'export.c',
+  'file-io.c',
+  'gconf-gnomeui.c',
+  'gconf-io.c',
+  'ghtml.c',
+  'ghtml-deprecated.c',
+  'idle-dialog.c',
+  'idle-timer.c',
```

Comment:
Is this file actually included in the automake build? I noticed in my timeout work yesterday that changes in this file didn't seem to trigger a rebuild. If it's not included in the automake build, perhaps this matter also affects the cmake build?

Maybe we have more files like that?

Reply:
Well spotted! I added both the `idle-timer`  header and source files to `src/Makefile.am`. For further missing files - I couldn't find any :-)